# distance between the camera mount and the center of the gripper. All units in milimeter
mount_translation_x: 0.0
mount_translation_y: 69.66
mount_translation_z: 41.1 

d405_height: 23.0
d405_center2left: 9.0

w: 1280
h: 720
fl_x: 1297.67
fl_y: 1304
cx: 620.91
cy: 238.28
k1: 0.0
k2: 0.0
p1: 0.0
p2: 0.0